src/inverseRobot.js

```diff
@@ -25,6 +25,17 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const inverse = {};
```

Comment:
Use clearer varaible names.

```suggestion
  const inversedRobot = {};
```